metrics/web/test/auth/domain/usercases/google_sign_in_usecase_test.dart

```diff
@@ -21,14 +40,108 @@ void main() {
       );
     });
 
+    test("throws if the repository.getGoogleSignInCredentials throws", () {
```

Comment:
```suggestion
    test("throws if the UserRepository throws during getting the google sign-in credentials", () {
```
I think we should not use references to the members of other classes(classes not under tests) in test descriptions because it makes them more complex and hard-to-read. I'll be thankful if you'll replace all such descriptions with more descriptive.